src/js/components/ServiceDetailConfigurationTab.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import ConfigurationView from './ConfigurationView';
+import Service from '../structs/Service';
+
+class ServiceConfigurationTab extends React.Component {
+
+  render() {
+    let {service} = this.props;
+    let {currentVersionID} = service.getVersionInfo();
```

Comment:
Yeah, there we will an extend version of the configuration tab to display not only the current but also older versions.